ocaml/idl/datamodel_repository.ml

```diff
@@ -18,6 +18,15 @@ open Datamodel_roles
 
 let lifecycle = [(Lifecycle.Published, "1.301.0", "")]
 
+let origin =
+  Enum
+    ( "origin"
+    , [
+        ("remote", "The repository will sync from reomte repository")
```

Comment:
"The repository will sync from reomte repository"
I think the datamodel `repository` for `remote` is the remote repository itself?